homeassistant/components/n26/__init__.py

```diff
@@ -42,8 +42,21 @@ def setup(hass, config):
     password = config[DOMAIN][CONF_PASSWORD]
     card_id = config[DOMAIN][CONF_CARD_ID]
 
+    from n26 import api, config as api_config
+    api = api.Api(api_config.Config(user, password, card_id))
+
+    from requests import HTTPError
+    try:
+        api._get_token()  # pylint: disable=W0212
```

Comment:
Please write out the disable name in words. I think this one is called `protected-access`.

Reply:
For the future, it would be good if the library exposed a method to check credentials intentionally. Eg by renaming `_get_token` to `get_token`.

Reply:
I will add that to the issue I just created. If the card id can be removed I'm pretty sure that I will be the one implementing it so I can do that in one PR (over there).